base/Dockerfile

```
@@ -6,54 +6,47 @@
 
-# This image is used to speed up build process for official netdata images.
-
-FROM alpine:3.18 as builder
+FROM debian:12 as builder
 
 LABEL org.opencontainers.image.authors="Netdatabot <bot@netdata.cloud>"
 LABEL org.opencontainers.image.source="https://github.com/netdata/helper-images"
 LABEL org.opencontainers.image.title="Netdata Agent Docker Base Image"
 LABEL org.opencontainers.image.description="Base image for official Netdata Agent Docker images."
 LABEL org.opencontainers.image.vendor="Netdata Inc."
 
-# Install some prerequisites
-RUN apk --no-cache add apcupsd \
-                       curl \
-                       fping \
-                       iproute2 \
-                       jq \
-                       json-c \
-                       libelogind \
-                       libgcrypt \
-                       libmnl \
-                       libuuid \
-                       libuv \
-                       libvirt-daemon \
-                       lm-sensors \
-                       lz4 \
-                       lz4-libs \
-                       mongo-c-driver \
-                       msmtp \
-                       ncurses \
-                       netcat-openbsd \
-                       nut \
-                       openssl \
-                       postgresql-client \
-                       protobuf \
-                       py3-pip \
-                       python3 \
-                       shadow \
-                       snappy \
-                       util-linux \
-                       yaml \
-                       zlib \
-                       zstd-libs && \
-    if [ "$(uname -m)" != "ppc64le" ]; then \
-        apk --no-cache add freeipmi freeipmi-libs || exit 1 ; \
-    fi
+ENV DEBIAN_FRONTEND=noninteractive
 
-# Add extra dependencies from alpine-edge
-#RUN apk --no-cache \
-#        --repository http://dl-cdn.alpinelinux.org/alpine/edge/community \
-#        --repository http://dl-cdn.alpinelinux.org/alpine/edge/main \
-#        add
+RUN chown _apt:root /var/cache/apt /var/lib/apt && \
+    apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y --no-install-recommends \
+                    curl \
+                    fping \
+                    iproute2 \
+                    jq \
+                    libgcrypt20 \
+                    libjson-c5 \
+                    liblz4-1 \
+                    libmariadb3 \
+                    libmnl0 \
+                    libmongoc-1.0-0 \
+                    libprotobuf32 \
+                    libsnappy1v5 \
+                    libssl3 \
+                    libsystemd0 \
+                    libuuid1 \
+                    libuv1 \
+                    libvirt0 \
+                    libyaml-0-2 \
+                    libzstd1 \
+                    lm-sensors \
+                    msmtp \
+                    ncurses-base \
+                    netcat-openbsd \
+                    openssl \
+                    python3 \
```

```suggestion
                    openssl \
                    procps \
                    python3 \
```

It would be nice to have `ps` command working, it really helps to debug things (usually the first command you execute when checking anything).